NetworkIOMeter.c

```diff
@@ -58,20 +56,24 @@ static void NetworkIOMeter_updateValues(Meter* this) {
       }
 
       cached_last_update = host->realtimeMs;
+   }
 
-      if (status == RATESTATUS_NODATA) {
-         xSnprintf(this->txtBuffer, sizeof(this->txtBuffer), "no data");
-         return;
-      }
+   if (hasNewData) {
+      static uint64_t cached_rxb_total = 0;
```

Comment:
Same comments as earlier (no init needed, move to struct) but this time for NetworkIOData.